classes/Language.php

```diff
@@ -1548,7 +1548,10 @@ public static function updateMultilangTable($iso_code)
             $rows = Db::getInstance()->executeS('SHOW TABLES LIKE \'' . str_replace('_', '\\_', _DB_PREFIX_) . '%\_lang\' ');
             if (!empty($rows)) {
                 // get all values
-                $tableNames = array_map('reset', $rows);
+                $tableNames = [];
```

Comment:
Why the `array_map` doesn't run ?

Reply:
Because `reset` needs a reference and it's not possible with `array_map` anymore.